app/controllers/ProjectSidewalkAPIController.scala

```diff
@@ -147,7 +147,9 @@ class ProjectSidewalkAPIController @Inject()(implicit val env: Environment[User,
       writer.println(header)
       // Write each row in the CSV.
       for (current <- GlobalAttributeTable.getGlobalAttributesWithLabelsInBoundingBox(minLat, minLng, maxLat, maxLng, severity)) {
-        writer.println(current.attributesToArray.mkString(","))
+        val rowArr: Array[String] = current.attributesToArray
+        rowArr(6) = "\"" + rowArr(6) + "\""
```

Comment:
Let's try our best not to rely on knowing which index in the array to use for the neighborhood name. Is there a more programmatic way that we can do it, like adding the quotes in the `attributesToArray` method?